fixtures/chauvet-dj/120st.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Follow Spot 120ST",
+  "shortName": "120ST",
+  "categories": ["Dimmer", "Other"],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2023-05-24",
+    "lastModifyDate": "2023-05-24"
+  },
+  "links": {
+    "manual": [
+      "https://www.chauvetdj.com/wp-content/uploads/2016/09/LED_Followspot_120ST_UM_Rev3.pdf",
+      "https://www.chauvetdj.com/wp-content/uploads/2016/09/LED_Followspot_120ST_QRG_Rev2_ML6_Print.pdf"
+    ],
+    "productPage": [
+      "https://www.chauvetdj.com/products/led-followspot-120st/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=mxAcqb5Cw1A"
+    ]
+  },
+  "physical": {
+    "dimensions": [279, 341, 582],
+    "weight": 6,
```

Comment:
Webpage says 7.5 kg, but the manual says 6, so, sure.

Reply:
I use the manual as my source when creating fixtures since the majority of the data regarding the fixture are there. I only turn to the websites when the data is not in the manual.